src/libs/auth.js

```diff
@@ -184,6 +190,16 @@ authStore.subscribe(async (state, oldState) => {
   }
 })
 
+authStore.subscribe(state => {
+  // We can't guarantee that someone reopening the window is the same user,
+  // so we should not persist cookie acceptance across sessions for people signed out
+  // Per compliance recommendation, we could probably persist through a session, but
+  // that would require a second store in session storage instead of local storage
+  if (state.isSignedIn && state.cookiesAccepted !== getLocalPref(cookiesAcceptedKey)) {
+    setLocalPref(cookiesAcceptedKey, state.cookiesAccepted)
```

Comment:
For users who stay signed in, does there need to be a cookie acceptance timeout for a session?

Reply:
There wasn't a requirement for that :D